src/azure-cli/azure/cli/command_modules/batch/_params.py

```diff
@@ -95,10 +101,12 @@ def load_arguments(self, _):
     with self.argument_context('batch application create') as c:
         c.argument('allow_updates', options_list=('--allow-updates',), action="store_true", help="Specify to indicate whether packages within the application may be overwritten using the same version string. True if flag present.")
 
-    with self.argument_context('batch application package create') as c:
-        c.argument('package_file', type=file_type, help='The path of the application package in zip format', completer=FilesCompleter())
-        c.argument('application_name', options_list=('--application-name',), help="The name of the application.")
-        c.argument('version_name', options_list=('--version-name',), help="The version name of the application.")
+    for command in ['create', 'activate']:
+        with self.argument_context('batch application package {}'.format(command)) as c:
+            c.argument('package_file', type=file_type, help='The path of the application package in zip format', completer=FilesCompleter())
+            c.argument('application_name', options_list=('--application-name',), help="The name of the application.")
+            c.argument('version_name', options_list=('--version-name',), help="The version name of the application.")
+            c.argument('f_ormat', options_list=('--format',), help="The format of the application package binary file.")
```

Comment:
```suggestion
            c.argument('format', options_list=('--format',), help="The format of the application package binary file.")
```

Reply:
Format is a keyword. If we use format as the parameter name, linter will report an error.

Reply:
You are right. Options_list is the display name. No problem.